README.md

```diff
@@ -1 +1,33 @@
-# Git_With_Girls_Summary
\ No newline at end of file
+# Git_With_Girls_Summary
+
+## Pragmatics
+
+Pragmatics is the study of how context affects meaning. It involves analyzing physical and linguistic context to understand communication.
```

Comment:
Pragmatics is the study of how context affects meaning **and how words can express things that are different from what they appear to mean.**